Library/Homebrew/cmd/update-report.rb

```diff
@@ -104,6 +105,36 @@ def update_report
       next unless tap.git?
       next if tap.core_tap? && ENV["HOMEBREW_INSTALL_FROM_API"].present? && args.preinstall?
 
+      # HOMEBREW_CORE_MERGE_MAINTAINER can be set by the maintainers
```

Comment:
Could maybe even pull this logic into a `LinuxbrewCoreMigration` class or something and call a single function there to do most of this? I dunno, maybe overkill.